site/content/en/docs/Installation/Creating Cluster/gke.md

```diff
@@ -143,16 +159,16 @@ gcloud compute firewall-rules create game-server-firewall \
 ```
 
 {{< alert title="Note" color="info">}}
-Before planning your production GKE infrastructure, it is worth reviewing the 
+Before planning your production GKE infrastructure, it is worth reviewing the
 [different types of GKE clusters that can be created](https://cloud.google.com/kubernetes-engine/docs/concepts/types-of-clusters),
 such as Zonal or Regional, as each has different reliability and cost values, and ensuring this aligns with your
-Service Level Objectives or Agreements. 
+Service Level Objectives or Agreements.
 
 This is particularly true for GKE masters, which can go down temporarily to adjust for cluster resizing,
```

Comment:
This is unrelated to your change, but since you are in this file....

Can you change "This is particularly true for GKE masters, which can go down temporarily" to "This is particularly true for a zonal GKE cluster control plane, which can go down temporarily"?